### PR TITLE
werf 1.2.87 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/werf.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  license "Apache-2.0"
+  head "https://github.com/werf/werf.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on :macos
+
+  def install
+    ldflags = "-s -w -X github.com/werf/werf/pkg/werf.Version=#{version}"
+    tags = "dfrunmount dfssh containers_image_openpgp"
+    tags << "osusergo exclude_graphdriver_devicemapper netgo no_devmapper static_build" if OS.linux?
```
[the remark this answers is not in the document]
Comment (carlocab) [reply Formula/werf.rb:15]: Upstream repo indicates Linux support tho

[the remark this answers is not in the document]
Comment (SMillerDev) [reply Formula/werf.rb:15]: In that case `depends_on :macos` should be removed

[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/werf.rb:15]: I will remove that tags operation, the linux support is complicated (as there are some header missing)

File: Formula/werf.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+class Werf < Formula
+  desc "Consistent delivery tool for Kubernetes"
+  homepage "https://werf.io/"
+  url "https://github.com/werf/werf/archive/refs/tags/v1.2.87.tar.gz"
+  sha256 "f008304325e96e6c7f28f68d49ffcc6e36ba26631d5a6bcd808c1b15d1d7c0d1"
+  license "Apache-2.0"
+  head "https://github.com/werf/werf.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on :macos
```
Comment (SMillerDev) [inline Formula/werf.rb:10]: Can you add a comment here to say that Linux support just hadn't been added yet? Since it doesn't actually depend on macOS

Comment (chenrui333) [reply Formula/werf.rb:11]: good call.

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks folks!

**Pull request closed without merging**